lib/velum/secrets.rb

```diff
@@ -0,0 +1,20 @@
+require 'securerandom'
+
+module Velum
+  # If there is a rails key base stored on disk, return the stored one.
+  # Otherwise, create a new key base from cryptographic random pool and store it.
+  def self.create_secret_key_base(pathname)
+    key_base = nil
+    if pathname.exist?
+      content = IO.read(pathname)
```

Comment:
I agree with your comment if we pass the `Pathname` around. I have never used `Pathname` since I think is a bit misleading in front of `File.read` (that for me is clear -> read this file), and `Pathname` semantics are a bit more blurry -- but I agree with your comments regarding `pathname` if we pass it around and use its instance methods.